typewriter/fixes/tests/test_util.py

```diff
@@ -0,0 +1,114 @@
+""" Test suite for the code in fixer_utils """
+
+# Local imports
+from lib2to3.fixer_util import (Attr, Call, Comma, Name, find_binding,
+                                is_import, syms)
+from lib2to3.pgen2 import token
+from lib2to3.pytree import Leaf, Node
+from lib2to3.tests.support import TestCase
+from lib2to3.tests.test_util import parse
+from typing import NamedTuple
+
+from .. import fixer_utils
+
+ImportInfo = NamedTuple("ImportInfo", [
+    # The bound name, i.e. "import x as y" would bind the name y for module x
+    ("binding_name", str),
+    ("imp_name", str),
+    ("node", Node)
+])
+
+
+class Test_get_import_binding_info(TestCase):
+    def _import_binding(self, node):
+        for c in node.children:
+            if is_import(c):
+                return c
+            return self._import_binding(c)
+
+    def get_import_binding_info(self, string, name, package):
+        node = parse(string)
+        # Return the node as an import binding if possible, None if otherwise
+        node = self._import_binding(node)
+        return fixer_utils._get_import_binding_info(node, name, package)
+
+    def try_with(self, string):
```

Comment:
if we do keep this TestCase class - it looks like this `string` arg is never used.